nimble/host/audio/services/bass/include/services/bass/ble_audio_svc_bass.h

```diff
@@ -286,6 +286,9 @@ struct ble_svc_audio_bass_operation {
             /** Number of subgroups */
             uint8_t num_subgroups;
 
+            /** BIS Synchronisation of subgroups */
```

Comment:
I think we could split it to small patches which address different problems.

Reply:
This is no longer present in current patch set.